contracts/contracts/ramps/wise/WiseSendProcessor.sol

```diff
@@ -0,0 +1,106 @@
+//SPDX-License-Identifier: MIT
+
+import { ECDSA } from "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import { SignatureChecker } from "@openzeppelin/contracts/utils/cryptography/SignatureChecker.sol";
+
+import { IKeyHashAdapterV2 } from "../../processors/keyHashAdapters/IKeyHashAdapterV2.sol";
+import { INullifierRegistry } from "../../processors/nullifierRegistries/INullifierRegistry.sol";
+import { IWiseSendProcessor } from "./interfaces/IWiseSendProcessor.sol";
+import { StringConversionUtils } from "../../lib/StringConversionUtils.sol";
+import { TLSBaseProcessor } from "../../processors/TLSBaseProcessor.sol";
+
+pragma solidity ^0.8.18;
+
+contract WiseSendProcessor is IWiseSendProcessor, TLSBaseProcessor {
+
+    using ECDSA for bytes32;
+    using SignatureChecker for address;
+    using StringConversionUtils for string;
+
+    /* ============ Constants ============ */
+    bytes32 public constant PAYMENT_STATUS = keccak256(abi.encodePacked("OUTGOING_PAYMENT_SENT"));
+
+    /* ============ Constructor ============ */
+    constructor(
+        address _ramp,
+        INullifierRegistry _nullifierRegistry,
+        uint256 _timestampBuffer,
+        string memory _endpoint,
+        string memory _host
+    )
+        TLSBaseProcessor(
+            _ramp,
+            _nullifierRegistry,
+            _timestampBuffer,
+            _endpoint,
+            _host
+        )
+    {}
+    
+    /* ============ External Functions ============ */
+    function processProof(
+        IWiseSendProcessor.SendProof calldata _proof
```

Comment:
All the other processors also accept proof. But in the case of send, proof also contains verifierSigningKey apart from the signature and public signals. Would advocate for keeping the passing in verifierSigningKey explicitly.